### ISSUE TITLE
Extension fails to activate on MacOS

### ISSUE THREAD

**Issue by coma-toast** [mondersky/tabscolor-vscode#6]
Looks to be similar to the following issue but different enough to warrant a new issue: https://github.com/mondersky/tabscolor-vscode/issues/3 

Same symptoms as above issue - the list of colors exist, but they do nothing. Seems to be related to the file path format. I have not yet tried on linux. 

```ERR [uncaught exception in main]: [UriError]: If a URI does not contain an authority component, then the path cannot begin with two slash characters ("//"): Error: [UriError]: If a URI does not contain an authority component, then the path cannot begin with two slash characters ("//")
    at b (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:17:83717)
    at new m (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:17:84358)
    at new h (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:17:86006)
    at h.with (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:17:85052)
    at M.asFileUri (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:18:4041)
    at m.requestToNormalizedFilePath (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:32:15598)
    at m.handleResourceRequest (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:32:15214)
    at Function.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/electron-main/main.js:32:14549)
```

<img width="531" alt="Screen Shot 2022-07-02 at 8 43 28 AM" src="https://user-images.githubusercontent.com/14114199/177001270-93d784fb-9907-497d-b214-56ea16a19b77.png">

**Comment by mondersky** [comment]
thanks for reporting the issue. The extension doesn't work on Mac OS unfortunately, I was unable to find a way to patch the vs code files in Mac OS

**Comment by ivnnv** [comment]
it is because you dont have a mac machine to try on? If you could point to where in the repo would need to try/modify im happy to help and create a PR if you provide some guidance.

**Comment by mondersky** [comment]
Hello, thanks a lot for your help, I've prepared two new vs commands you can use to debug the problem:
1 tabsColor debug: locate target file
This will give you the location of the read-only file that I need to allow writing to.
2 tabsColor debug: debug bootstrap file on mac
This will execute the js instructions that are supposed to allow access to the target file, This is where you should write your code
https://github.com/mondersky/tabscolor-vscode/blob/082fb76959d11f70ae2849fdf3e8c2c1a528bf1a/extension.js#L439

In order to test your code:
- uninstall the extension from your vs code
- go to the extension folder and run "vsce package" this will generate a vsix file there
- go to vs code > extension and click the tree dots icon on top > install from VSIX and chose the file you previously generated
- run the command "tabsColor debug: debug bootstrap file on mac" on vs code to run your code

Basically if you manage to make the file bootstrap-window.js writable from within the extension this means that you have found a way to solve the problem.

**Comment by kobilee** [comment]
I found a workaround for this issue, moving your installation of vscode into your application folder will allow bootstrap-window.js to become writable from within the extension

<img width="664" alt="Screen Shot" src="https://user-images.githubusercontent.com/43255165/235240380-e0339b7b-0d42-4010-8e90-d341b10cfb9a.png">

**Comment by McDusty** [comment]
is this something a user can manually do until this is officially supported by the plugin? I am a mac user and I want this feature so bad.

**Comment by kobilee** [comment]
I have spent many hours trying to Implement this feature without user intervention and I cannot find any way to modify the permissions on those protected files to inject the required code. 

I would guess that unless/until the vscode API give access to modifying the styles on components this manual workout around will be the only way for Mac users. 

I would not hold your breath for a non-manual solution to be supported on Mac anytime soon

**Comment by dotdev-nikolaj** [comment]
@kobilee My vscode is in my Applications folder? :O 

<img width="699" alt="Screenshot 2023-10-03 at 3 13 57 pm" src="https://github.com/mondersky/tabscolor-vscode/assets/134979630/64fe9b26-9e5c-4850-9952-05978d9fda7c">
